src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -210,6 +210,10 @@ export class CellModel implements ICellModel {
 				await kernel.interrupt();
 			} else {
 				// TODO update source based on editor component contents
+				if (kernel.requiresConnection && !this.notebookModel.activeConnection) {
+					this.sendNotification(notificationService, Severity.Error, localize('kernelRequiresConnection', 'Please select a connection to run cells for this kernel'));
```

Comment:
double quotes. Maybe put a period at the end?

Reply:
Double-quotes yes. Period at the end, no. It's not needed for an error message and I want to minimize cases where the message gets an expandy-arrow by keeping it short.